### PR TITLE
Decklist expander

### PR DESCRIPTION
**Pull request by hssm**
First commit is a fix for [Issue 2696](https://code.google.com/p/ankidroid/issues/detail?id=2696).

Second one replaces the deck expander text with a drawable instead. I had to take a different approach to the padding for deck nesting, so the widths aren't *exactly* the same, but I tried to make it as similar as possible. Some before/after images to compare:

expander_day
expander_night

I also removed the context menu expand/collapse since it is not really needed with this method around.

### CODE REVIEW

**Comment by hssm** [comment]
Also looks fine on a tiny screened 2.2 device.

device-2015-07-26-135527

File: AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java [hunk 1]
```diff
@@ -103,7 +117,11 @@ public DeckAdapter(LayoutInflater layoutInflater, Context context) {
         mRowCurrentColor = ta.getColor(5, res.getColor(R.color.deckadapter_row_current));
         mDeckNameDefaultColor = ta.getColor(6, res.getColor(R.color.black));
         mDeckNameDynColor = ta.getColor(7, res.getColor(R.color.deckadapter_deck_name_dyn));
+        mExpandImage = ta.getDrawable(8);
+        mCollapseImage = ta.getDrawable(9);
```
Comment (timrae) [inline AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java]: It looks like this isn't used anywhere...?

File: AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java [hunk 2]
```diff
@@ -187,31 +206,28 @@ public int getItemCount() {
...
+        // Apply the correct expand/collapse drawable
         if (collapsed) {
-            // add arrow pointing right if collapsed
-            s = "\u25B7 ";
-        } else if (children > 0) {
-            // add arrow pointing down if deck has children
-            s = "\u25BD ";
+            expander.setImageDrawable(mExpandImage);
+        } else if (node.children.size() > 0) {
+            expander.setImageDrawable(mCollapseImage);
```
Comment (hssm) [inline AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java]: :tada:

File: AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java (hunk 1 above)
Comment (hssm) [inline AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java]: It's used below.

Comment (timrae) [inline AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java]: Ah sorry I was just using the chrome search function and there was an extra space at the end of the word so it only had one match lol. Also there were no right-facing-arrows in the screenshot which was a bit confusing ;)

**Pull request merged**